docs/docs/deploy/advanced/reverse-proxy.mdx

```diff
@@ -1,20 +1,24 @@
 ---
 title: Custom Reverse Proxy
-sidebar_position: 4
+sidebar_position: 3
 ---
 
-:::note
+:::warning
 Using a custom reverse proxy is an advanced configuration. The default bundled
-Nginx proxy is suitable for the vast majority of use cases and is recommended
-for most users. There are important security risks if the Firezone reverse proxy
-is not set up correctly. Use only if you know what you are doing.
+Nginx proxy (Omnibus-based deployments) and Caddy (Docker-based deployments) is
+suitable for the vast majority of use cases and is recommended for most users.
+There are important security risks if the reverse proxy is not set up correctly.
 :::
 
+For help deploying Firezone with a custom reverse proxy, consider [contacting us
+for support](https://www.firezone.dev/contact/sales).
+
 ## Introduction
 
-Firezone comes with a bundled [Nginx](https://www.nginx.com/) reverse proxy,
-however, in some cases you might want to deploy your own server such as when
-using behind your own load balancer.
+Firezone comes with bundled [Nginx](https://www.nginx.com/) (Omnibus-based
+deployments) or uses Caddy (Docker-based deployments) by default. However, in
+some cases you might want to deploy your own server such as when using behind
```

Comment:
Maybe just remove `behind` here. It's a little awkward with it and without it the sentence still makes sense.